api/config/v1/flags.go

```diff
@@ -59,6 +59,7 @@ type CommandLineFlags struct {
 	FailOnInitError   *bool                   `json:"failOnInitError"            yaml:"failOnInitError"`
 	MpsRoot           *string                 `json:"mpsRoot,omitempty"          yaml:"mpsRoot,omitempty"`
 	NvidiaDriverRoot  *string                 `json:"nvidiaDriverRoot,omitempty" yaml:"nvidiaDriverRoot,omitempty"`
+	HostDevRoot       *string                 `json:"hostDevRoot,omitempty"      yaml:"hostDevRoot,omitempty"`
```

Comment:
Thoughts on naming this field `NvidiaDevRoot` to closer align with `NvidiaDriverRoot`?

Reply:
Yes, I think we could. Will update.